public/src/components/campaign/joined_campaigns.jsx

```diff
@@ -53,6 +53,7 @@ export default class JoinedCampaigns extends Component {
               <CurrentCampaign
                 history={history}
                 userId={userId}
+                list={joined_campaigns}
                 currentCampaign={current_campaign}
                 defaultMsg={joined_campaigns && joined_campaigns.length > 0 ?
```

Comment:
I think the defaultMsg can just be set to "Join a campaign"

Reply:
There were a couple of weird scenarios where "Select a Campaign" was still popping up, so I think I'll keep it for now if that's okay with you.